plugins/MauticCrmBundle/Integration/Pipedrive/Import/LeadImport.php

```diff
@@ -63,6 +62,7 @@ public function update(array $data = [])
             return $this->create($data);
         }
 
+        /** @var $lead LeadModel */
         $lead         = $this->em->getRepository(Lead::class)->findOneById($integrationEntity->getInternalEntityId());
```

Comment:
@kuzmany are you sure the $lead variable is instance of LeadModel? Shouldn't it be instance of the Lead entity? If so, update the annotation and remove the LeadModel namespace.

Reply:
Yeah, thats true. Fixed